### DISCUSSION TITLE
Use different positions Notification system

### DISCUSSION THREAD

**Discussion by realyuki** [mantinedev/mantine#6114] [Q&A]
Hi. Thank you for your help as always.

I want to create a Toast component using Notification system.
Although I set the provider's position differently each time using useState, I want these two components to stack in different positions.

However, the alignment of Toast must be "top-center", and the alignment of Notification must be "bottom-right". Can I set these two positions differently using NotificationProvider?

**Comment by rtivital** [comment]
Currently, it is not documented, you can browse source code to learn how to setup multiple notifications systems in your application.

**Comment by tombohub** [comment]
Can you give me an example how to use your custom component in notification system? I cannot figure it out
